### PR TITLE
Add product vision to README

### PR DESCRIPTION
**Pull request by jordynhayden**

### CODE REVIEW

**Comment by jordynhayden** [comment]
Hey team, here is my initial draft of the product vision. Please provide some feedback and offer suggestions on how we can improve it to meet the guidelines.

1. For ... (the product’s target market)
2. Who/That ... (describe what need your product solves)
3. The product ... (product name, or product category)
4. That leads to ... (main advantage, appealing reason for purchase)
5. Unlike the ... (principal competing alternatives)
6. Our product offers ... (describe main differentiation)

@jcoolrace @jaerom @diegoro1 @BlackOutDevelops

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Clippy
 
+# Product Vision
+
+For internet users who find themselves having a difficult time organizing and discovering resources on the Web that sparks their interest, Clippy is a social bookmarking service designed for users to explore and manage resources on the Internet. Users can immerse themselves in their own distraction-free space curated by them or explore resources gathered by others in the community. Unlike many other social bookmarking services, such as Pocket, Clippy is currently a free service made available to everyone.
+
 ## Team Members
 
 - Joshua Frazer
```
Comment (BlackOutDevelops) [inline README.md:9]: The first sentence is kind of repetitive and is a run on sentence. Not sure if he cares about grammar and such. I'd revise it to: "For internet users who find themselves having a difficult time organizing and discovering resources on the Web that spark their interest, we introduce Clippy." Then for what comes after the coma, "Clippy is a social bookmarking service in which users can immerse themselves in their own distraction-free space curated by themselves or share with others in the community."
The "That leads to... (main advantage, appealing reason for purchase)" list item he has, our product is free. So I doubt he would have issues, but not sure to be honest.
 All in all, great job though. Thanks for starting us off, Jordyn.
By the way, forgive me, never used this before, still trying to learn.

Comment (jaerom) [reply README.md:9]: +1 The above changes.

I think guidelines 4 through 6 are fine the way they are. Does this final version bellow look good?

**Final Version**
For internet users who find themselves having a difficult time organizing and discovering resources on the Web that spark their interest, we introduce Clippy. Clippy is a social bookmarking service in which users can immerse themselves in their own distraction-free space curated by themselves or share with others in the community. Unlike many other social bookmarking services, such as Pocket, Clippy is currently a free service made available to everyone.

**Review by diegoro1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jordynhayden) [reply README.md:9]: Great suggestions I'll push the changes. :)

**Pull request merged**